<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGameSettings</class>
 <widget class="QDialog" name="DialogGameSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QPushButton" name="normalGameButton">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>250</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="infiniteGameButton">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>250</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string> Infinite time game</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>-10</y>
     <width>200</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Overpass</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nysse game</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>180</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Gamemode: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>140</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter player name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>140</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox1min">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>1min game</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox2min">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>2min game</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>471</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome to game called Nysse! 
Game goal is to drive taksi in Tampere and &quot;pick up&quot; passengers. You 
can choose the player name and time limit for the game below.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
